app/scripts/directives/blocks.js

```diff
@@ -111,7 +111,12 @@ angular.module('confRegistrationWebApp')
   .directive('dateQuestion', function () {
     return {
       templateUrl: 'views/blocks/dateQuestion.html',
-      restrict: 'E'
+      restrict: 'E',
+      controller: function($scope){
+        if(angular.isDefined($scope.answer) && !($scope.answer.value instanceof Date)){
+          $scope.answer.value = new Date($scope.answer.value);
```

Comment:
No. That was to fix the errors with the HTML5 date picker since the angular input date directive requires the model to be a date object.